### PR TITLE
Donald hager

### PR DESCRIPTION
**Pull request by b1zz3**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,6 +73,7 @@ <h3 class="spotlight-subtitle">
               <a href="#" class="button button-spotlight">Grab the app!</a>
             </span>
             
+>>>>>>> c873ada5adf93644d73d697b10fb36b75f202cc8
```
Comment (ssmodish) [inline index.html]: This looks like the remnants of a merge conflict.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        </div>
+
+    </body>
+
+    <footer id="footer" class="footer">
+        <p>© Copyright D'Salty Dogs of LS</p>
+        <p>Social Media Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></p>
+    </footer>
+
+</html>
```
Comment (ssmodish) [inline about.html:176]: I don't think this is an issue, but Github has it marked in red so it might be something

File: about.html [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                        <div class="member-name">    
+                            <h1>Donald Hager</h1>
+                            <h2>Full Stack</h2>
+                        </div>
+                    </div>
+                    <ul class="soc-media">
+                        <li><img class="soc-logo" src="images/Team/logos/github.png"> <a href="https://github.com/b1zz3">github.com/b1zz3</a></li>
+                        <!-- <li><img class="soc-logo" src="images/Team/logos/twitter.png"> <a href=" --link here-- "> --link here -- </a></li> -->
+                        <!-- <li><img class="soc-logo" src="images/Team/logos/linkedin.png"> <a href=" --link here-- "> --link here -- </a></li> -->
+                        <li><img class="soc-logo" src="images/Team/logos/gmail.png"> <a href="donhager23@gmail.com">donhager23@gmail.com</a></li>
```
Comment (ssmodish) [inline about.html:144]: Just use `href="#"` when the link hasn't yet been decided

**Review by ssmodish — CHANGES_REQUESTED** [review]

File: about.html (hunk 2 above)
Comment (b1zz3) [reply about.html:176]: I'm not sure about this one either, i'll just get rid of the link and leave the text credit.

Comment (mikepadiernos) [reply about.html:176]: I think github is assuming the "www.flaticon.com" between the a tag is a broken link. Maybe replace it with just Flaticon.

File: about.html [hunk 4]
```diff
@@ -133,25 +126,21 @@ <h2>Full Stack</h2>
...
-                        <!-- <li><img class="soc-logo" src="images/Team/logos/linkedin.png"> <a href=" --link here-- "> --link here -- </a></li> -->
                         <li><img class="soc-logo" src="images/Team/logos/gmail.png"> <a href="mike.padiernos@gmail.com">mike.padiernos@gmail.com</a></li>
                     </ul>  
                 </div>
                 
                 <!-- Donald Hager -->
                 <div class="team-card">
                     <div class="tCard-top">
-                        <img class="member-img" src="images/Team/martinLehman.png">
+                        <img class="member-img" src="images/Team/donaldHager.jpg">
```
Comment (ssmodish) [inline about.html:136]: 🤣 How did we all miss this one!

**Review by ssmodish — APPROVED** [review]

**Pull request merged**